shared/src/validation-schemas/group/group-create-name.validation-schema.ts

```diff
@@ -0,0 +1,23 @@
+import Joi from 'joi';
+
+import {
+  GroupValidationMessage,
+  GroupValidationRule,
+} from '~/common/enums/enums';
+import { GroupsCreateRequestDto } from '~/common/types/types';
+import { getNameOf } from '~/helpers/helpers';
+
+const groupCreateName = Joi.object({
```

Comment:
```suggestion
const groupCreate = Joi.object({
```
don't forget about the name

Reply:
Yep, but such validation scheme already exists. That includes Ids arrays to validate (that we do not do because we moved useForm logic only on name field). Should I name it in some other way?

Reply:
May be I should make that schema only on frontend?

Reply:
And if I use the schema, which already exists - I won't be able to submit form

Reply:
yes, in this case, it should be `groupCreateClient`